### ISSUE TITLE
Latest artist slideshow plugin not supported

### ISSUE THREAD

**Issue by coldspark29** [jurialmunkey/skin.aura#89]
There is a new feature in the plugin, which is not supported. There is a plugin notification telling to inform the developer of the skin

**Comment by chucksplatt** [comment]
I just happened to have this fixed locally and was thinking it would be good to push out to the wider world, and then I saw your request on the top of the pile!

**Comment by chucksplatt** [comment]
Looks like the version of the skin in the Kodi repo is 1.0.2, while the one in the master branch is currently labelled 1.0.5 and has what is supposedly the fix for artist slideshow.  I just checked out the branch and am about to test it.  Should 1.0.5 get deployed to the Kodi repo?

**Comment by drinfernoo** [comment]
If @jurialmunkey doesn't intend on it being submitted to the official repo, I could see if he minds me adding it to his repo 😁

**Comment by chucksplatt** [comment]
Looks like artist slideshow should work fine as of 1.0.5/what's current on master

**Comment by coldspark29** [comment]
Yep, tried it out. It works. It should be pushed to the Kodi repo

**Comment by freijon** [comment]
I just wanted to point out that the current version in the repo is still 1.0.2 and that the artist slideshow feature is still unusable. Is this new version coming any time soon? Is there a timeline for this or something?

**Comment by drinfernoo** [comment]
@freijon 1.0.5 is in @jurialmunkey's repo.

**Comment by freijon** [comment]
Oh... I'm sorry, I just installed the skin from the standard Kodi repo. Where can I find jurialmunkey's repo?

**Comment by drinfernoo** [comment]
@freijon https://jurialmunkey.github.io/
